lib/datadog/statsd/sender.rb

```diff
@@ -10,7 +10,8 @@ def initialize(message_buffer)
       end
 
       def flush(sync: false)
-        raise ArgumentError, 'Start sender first' unless message_queue
+        # don't try to flush if there is no message_queue instantiated
+        return unless message_queue
```

Comment:
Seems like a good place for a warning though - I'm guessing this is not an expected situation

Reply:
It also means there is nothing to flush.
I agree a warning message may have been better (thought about it as well) but since: the logger is not accessible in this class with current code + nearly no chance to have anything to flush, I prefer to just fix the behavior for now.